include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (1)
+#define OPENTHREAD_API_VERSION (2)
```

Comment:
@bukepo what would be expected if multiple PRs update API version? 

Like the other one in [#4900](https://github.com/openthread/openthread/pull/4900/commits/76be02688ceb79b98c26a2113c18ab98494d969f#diff-68d9781e82e4802a218d26d0595157a7R56)

Reply:
I think we need to simply flag these during the review process (e.g. add the `api: user` and `api: platform` labels) and ensure such PRs are up-to-date with latest master before merging.